frontend/projects/upgrade/src/app/core/segments/store/segments.effects.ts

```diff
@@ -48,7 +48,7 @@ export class SegmentsEffects {
           actionType === UpsertSegmentType.CREATE_NEW_SEGMENT
             ? this.segmentsDataService.createNewSegment(Segment)
             : actionType === UpsertSegmentType.IMPORT_SEGMENT
-            ? this.segmentsDataService.importSegment(Segment)
+            ? this.segmentsDataService.importSegments([])
```

Comment:
Should this `importSegments()` be empty?
As we are passing Segment is `create` and `update`

Reply:
that's okay, I have created separate func for import so this won't be used when we import segments.